go.mod

```diff
@@ -22,24 +22,24 @@ require (
 	github.com/openstack-k8s-operators/openstack-baremetal-operator/api v0.3.1-0.20240222051903-0f74dcdc7782
 	golang.org/x/exp v0.0.0-20240213143201-ec583247a57a
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.28.7
-	k8s.io/apimachinery v0.28.7
-	k8s.io/client-go v0.28.7
+	k8s.io/api v0.29.0
```

Comment:
Should not this be  pinned to '<0.29.0'

https://github.com/openstack-k8s-operators/renovate-config/blob/main/default.json5#L35

Reply:
yes, this is also problematic